src/terminal/parser/OutputStateMachineEngine.hpp

```diff
@@ -160,6 +160,7 @@ namespace Microsoft::Console::VirtualTerminal
             SetWindowProperty = 3, // Not implemented
             SetColor = 4,
             Hyperlink = 8,
+            SetTaskbarProgress = 9,

```

Comment:
I know I'm sorta asking too much from this PR. But technically `9` should be `ConEmuAction` or `ConEmuCustom`, and the subparam `4` decides that the action is to `SetTaskbarProgress` .  This just suddenly occurs to me when I finished writing up in #8166 .

Reply:
Thank you for pointing that out!

For now, I've done a quick fix that skips the parsing step and just sends the string to the dispatcher to figure out.